app/views.py

```diff
@@ -30,14 +30,14 @@ def isLoggedIn():
     return True
 
 
-def getUser():
+def getUser(username):
     if not isLoggedIn():
         return {'error': "Not logged in."}
 
     ts = pymesync.TimeSync(baseurl=app.config['TIMESYNC_URL'],
                            test=app.config['TESTING'],
                            token=session['token'])
-    user = ts.get_users(username=session['username'])
+    user = ts.get_users(username=username)
 
     return user
```

Comment:
@aaroncohen73 What if we need to flash the error message to the user? By returning an empty dict we have no idea what went wrong.